tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -64,6 +64,10 @@ def test_rb_command_with_force_deletes_objects_in_bucket(self):
             with mock.patch(arch_name):
                 rb_command._run_main(parsed_args,
                                      parsed_globals=parsed_globals)
+                with self.assertRaises(ValueError):
```

Comment:
It would be nice if we had the assertion in a separate test. We typically try to avoid testing different functionality in the same test. So I would recommend adding a new test for this specific assertion in this test class or adding a functional test in this [directory](https://github.com/aws/aws-cli/tree/develop/tests/functional/s3) (Note that you would have to create your own ``test_rb_command.py`` file for the ``rb`` command and sort of copy the what the other tests do in that directory if you go down this route)